website/docs/reference/administrative-commands/createIndexes.mdx

```diff
@@ -0,0 +1,123 @@
+---
+sidebar_position: 1
+description: Create one or more indexes on a collection to improve query performance and enable certain query features.
+---
+
+# `createIndexes`
```

Comment:
![CleanShot 2025-05-26 at 23 28 38@2x](https://github.com/user-attachments/assets/0dc31803-dd7f-4c8f-8c7b-ee1c7a034f9e)

This looks ugly. Drop backticks.